docs/reST/ref/image.rst

```diff
@@ -202,10 +212,21 @@ following formats.
 
    .. note:: :func:`fromstring` and :func:`tostring` are named with "string,"
              but they deal in bytes. In Python 2, str = bytes, so this is
-             "from string" as in a Python 2 string.
+             "from string" as in a Python 2 string. The use of :func:`frombytes`/
+             :func:`tobytes` is preferred over :func:`fromstring`/:func:`tostring`
+             as of pygame 2.1.3.
 
    .. ## pygame.image.fromstring ##
 
+.. function:: frombytes
+
+   | :sl:`create new Surface from a string buffer`
+   | :sg:`frombytes(bytes, size, format, flipped=False) -> Surface`
+
+   since version 2.1.3 it is recommended that `frombytes` is used instead of `fromstring` 
```

Comment:
It would be better to mention with more clarity that this is an alias of `fromstring`.

Also needs the `.. versionadded::` tag